zerver/management/commands/change_user_role.py

```diff
@@ -86,9 +88,17 @@ def handle(self, *args: Any, **options: Any) -> None:
             print(
                 f"{user.delivery_email} changed to {granted_text} {options['new_role']} permission."
             )
-        else:
+        elif options["new_role"] == "can_create_users":
             if user.can_create_users and options["grant"]:
                 raise CommandError("User can already create users for this realm.")
             elif not user.can_create_users and not options["grant"]:
                 raise CommandError("User can't create users for this realm.")
             do_change_can_create_users(user, options["grant"])
+        else:
+            assert options["new_role"] == "is_billing_admin"
+            if user.is_billing_admin and options["grant"]:
+                raise CommandError("User already is a billing admin for this realm.")
+            elif not user.is_billing_admin and not options["grant"]:
+                raise CommandError("User is not a billing admin for this realm.")
+
+            do_change_is_billing_admin(user, options["grant"])
```

Comment:
This part looks great, opened as https://github.com/zulip/zulip/pull/27729 so it can merge without delay and shrink this a bit.